### PR TITLE
fixed progress calculation

### PR DESCRIPTION
**Pull request by smelukov**
**What kind of change does this PR introduce?**

bugfix

I've set `entries` to true and got `68% building 198/804 entries 2050/2115 modules 65 active .....`
198/804 is not 68%

**Did you add tests for your changes?**

no

**Does this PR introduce a breaking change?**

no

**What needs to be documented once your changes are merged?**

no

### CODE REVIEW

**Comment by smelukov** [comment]
Or maybe we should do something like:
```
cons progress = showEntries ? Math.min(percentByModules, percentByEntries) : percentByModules;
```

File: lib/ProgressPlugin.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class ProgressPlugin {
 					doneEntries / Math.max(lastEntriesCount, entriesCount);
 
 				const items = [
-					0.1 + Math.max(percentByModules, percentByEntries) * 0.6,
+					0.1 + Math.min(percentByModules, percentByEntries) * 0.6,
```
[the remark this answers is not in the document]
Comment (smelukov) [reply lib/ProgressPlugin.js]: Seems like the solution with mode-property was a better solution, cause we had posibility to choose calculation mode depending on a case

[the remark this answers is not in the document]
Comment (sokra) [reply lib/ProgressPlugin.js]: Did you try to pass a higher values to `modulesCount` that matches your module count?

[the remark this answers is not in the document]
Comment (smelukov) [reply lib/ProgressPlugin.js]: Yes, but it does not solve the problem and seems like a hack. Amount of the modules in our project is changing very often, cause we have really huge code base and many teams that changing it. In this case we should change progress plugin options often too :(

**Comment by smelukov** [comment]
@sokra, sorry about the mention, but what do you think about returning the mode-option?
The original idea was about choosing calculation mode for more accuracy percent value, not only about displaying entries count

**Comment by vankop** [comment]
Thanks for your contribution. I am closing PR because it is too stale..

**Pull request closed without merging**